src/components/Overview/Overview.react.js

```diff
@@ -196,19 +197,47 @@ class Overview extends Component {
               <p className="description__text">
                 <b>SUSI.AI</b> already runs on many chat services and social
                 networks. We are developing plugins for all major services
-                including &nbsp;<a href="https://github.com/fossasia/susi_tweetbot">
```

Comment:
this should be https://github.com/fossasia/susi_tweetbot   
please change the link 👍

Reply:
Updated